web/packages/teleport/src/Discover/useDiscover.tsx

```diff
@@ -226,9 +228,9 @@ export function DiscoverProvider({
   // The location.state.discover should contain all the state that allows
   // the user to resume from where they left of.
   function resumeDiscoverFlow() {
-    const { discover, integrationName } = location.state;
+    const { discover, integration } = location.state;
 
-    updateAgentMeta({ integrationName } as DbMeta);
+    updateAgentMeta({ integration } as DbMeta);
```

Comment:
The type system isn't complaining but this cast doesn't seem like it should work without making `db` and `selectedAwsRdsDb` optional.

```ts
export type DbMeta = BaseMeta & {
  // TODO(lisa): when we can enroll multiple RDS's, turn this into an array?
  // The enroll event expects num count of enrolled RDS's, update accordingly.
  db?: Database;
  integration?: Integration;
  selectedAwsRdsDb?: AwsRdsDatabase;
  // serviceDeployedMethod flag will be undefined if user skipped
  // deploying service (service already existed).
  serviceDeployedMethod?: ServiceDeployMethod;
};
```